scripts/add-cluster.sh

```diff
@@ -43,29 +51,24 @@ rules:
 - apiGroups:
   - toolchain.dev.openshift.com
   resources:
-  - "*"
```

Comment:
Why not using *?
We will have to update this file every time we add a new CRD if we don't use *. Which I'm sure we will forget to do.

Reply:
Because dedicated-admin doesn't have the `*` permission, so it cannot grant permission that doesn't hold. 

I know that it's not really flexible, but I'll try to change it in the future. I think that it would be good to rewrite it to go command where we can set the resources dynamically

Reply:
Ok. Not super critical right now but would be great to load them dynamical. For now let's add a comment to the make file in the api repo? The place where we list the kinds for make generate.

Reply:
This is the make file we can add a comment to for now: https://github.com/codeready-toolchain/api/blob/master/make/generate.mk#L6-L8

Reply:
https://github.com/codeready-toolchain/api/pull/203